[2024-12-06T23:30:33.557+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T23:30:33.566+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mediaset_schedule_scraper.PushAllInOne scheduled__2024-12-05T23:30:00+00:00 [queued]>
[2024-12-06T23:30:33.571+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mediaset_schedule_scraper.PushAllInOne scheduled__2024-12-05T23:30:00+00:00 [queued]>
[2024-12-06T23:30:33.571+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 5
[2024-12-06T23:30:33.578+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): PushAllInOne> on 2024-12-05 23:30:00+00:00
[2024-12-06T23:30:33.582+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5441) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T23:30:33.583+0000] {standard_task_runner.py:72} INFO - Started process 5442 to run task
[2024-12-06T23:30:33.583+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'mediaset_schedule_scraper', 'PushAllInOne', 'scheduled__2024-12-05T23:30:00+00:00', '--job-id', '2004', '--raw', '--subdir', 'DAGS_FOLDER/dag_mediaset.py', '--cfg-path', '/tmp/tmp4q24m727']
[2024-12-06T23:30:33.585+0000] {standard_task_runner.py:105} INFO - Job 2004: Subtask PushAllInOne
[2024-12-06T23:30:33.608+0000] {task_command.py:467} INFO - Running <TaskInstance: mediaset_schedule_scraper.PushAllInOne scheduled__2024-12-05T23:30:00+00:00 [running]> on host b605548c7b8b
[2024-12-06T23:30:33.646+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mediaset_schedule_scraper' AIRFLOW_CTX_TASK_ID='PushAllInOne' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T23:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-05T23:30:00+00:00'
[2024-12-06T23:30:33.647+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T23:30:36.010+0000] {logging_mixin.py:190} INFO - Archivo actualizado correctamente.
[2024-12-06T23:30:36.011+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T23:30:36.017+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T23:30:36.017+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=mediaset_schedule_scraper, task_id=PushAllInOne, run_id=scheduled__2024-12-05T23:30:00+00:00, execution_date=20241205T233000, start_date=20241206T233033, end_date=20241206T233036
[2024-12-06T23:30:36.044+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T23:30:36.053+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-06T23:30:36.054+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
